spec/serializers/basic_post_serializer_spec.rb

```diff
@@ -16,5 +16,20 @@
       SiteSetting.enable_names = false
       expect(json[:name]).to be_blank
     end
+
+    describe "#cooked" do
+      it "returns the post's cooked" do
+        expect(json[:cooked]).to eq(post.cooked)
+      end
+
+      it "returns the modified cooked when register modified" do
```

Comment:
for these specs I think you need to cleanup the modifier like thissss -> https://github.com/discourse/discourse/blob/8d709aeb9c8509f55aa5dabbc2665aac7da88c88/spec/services/push_notification_pusher_spec.rb#L211-L238